docs/resources/token.md

```diff
@@ -30,6 +30,22 @@ output "databricks_token" {
 }
 ```
 
+A token can be automatically rotated by taking a dependency on the `time_rotating` resource:
+
+```hcl
+resource "time_rotating" "this" {
+  rotation_months = 30
```

Comment:
per common on  PAT this should be `rotation_days` instead of `rotation_months`

Reply:
Whoops, thank you!